charts/ark-cloudconfig/templates/storageclass.yaml

```diff
@@ -1,11 +1,13 @@
+{{- if (eq 1 0) -}}
```

Comment:
Should you add some comment here, to remind yourself to remove this `eq 1 0` when you're ready to include the rest of this code?  I suppose this first line is meant to force-skip all the others.

Reply:
I'm actively working on this chart right now. It won't be functional before tomorrow as I need to talk to Nadica to get a good understanding of what this chart needs to look like. Lots of configuration crap, not to mention the shared filesystem that's required.